notebook/save_message_handler.js

```diff
@@ -0,0 +1,24 @@
+define([
+    'base/js/namespace'
+], function(
+    Jupyter
+) {
+    function load_handle_save_notebook_extension() {
+        window.addEventListener("message", (event) => {
+            const source = iframe_source || "http://localhost";
```

Comment:
it's always set no?

Reply:
True, but I don't know that in this place, just an additional guard